lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -1703,12 +1703,12 @@
                         platform.raiseNeedObjectOfType("DateTimeFormat.prototype.format", "DateTimeFormat");
                     }
 
-                    if (isNaN(date) || !isFinite(date)) {
+                    if (date !== undefined && (isNaN(date) || !isFinite(date))) {
                         platform.raiseInvalidDate();
                     }
 
                     let dateValue = undefined;
-                    if (date) {
+                    if (date !== undefined) {
```

Comment:
What happens if I pass empty string for date? Is it expected to be same as passing 0?

Reply:
the WinGlob version of this function implements a slightly different spec but https://tc39.github.io/ecma402/#sec-datetime-format-functions shows that we can work with the argument if its provided or not undefined.